### DISCUSSION TITLE
Components should not block for a long period of time

### DISCUSSION THREAD

**Discussion by jklap** [emporia-vue-local/esphome#196] [Q&A]
Recent change in esphome enabling logging:

https://github.com/esphome/issues/issues/4717#issuecomment-1682633629

**Comment by ForceConstant** [comment]
Is there a workaround for this issue for the vue2?

**Comment by roycamp** [comment]
also seeing this, can it be safely ignored or is it impacting something?

**Comment by clowrey** [comment]
its normal for this type of component - ignore or attempt to optimize the code if you're a programmer.
